### PR TITLE
tests: Cleanup - move common code

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by brybacki** [comment]
no idea what is going go with the test, let me retry
/test pull-kvp-functional-test

File: tests/framework/storage.go [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+		return false, err
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return dataVolume, nil
+}
+
+// FindPVC Finds the passed in PVC
```
Comment (alromeros) [inline tests/framework/storage.go:43]: Nitpick, but not too sure if `passed in` makes sense in this context. This is repeated several times along the PR.

**Review by alromeros — COMMENTED** [review]
Looks good! I like the changes. This PR is hard to review, so it'd be cool to have more reviewers.
/lgtm

**Comment by brybacki** [comment]
> Looks good! I like the changes. This PR is hard to review, so it'd be cool to have more reviewers. /lgtm

90% of work is just cut and paste some functions from one place to another, the rest is just using the functions

**Review by alromeros — COMMENTED** [review]
/lgtm

**Pull request merged**